### ISSUE TITLE
The "ov5.0.0" Dashboard builds but core dumps at startup (1.5.11rc2 specific)

### ISSUE THREAD

**Issue by canne** [canne/dashboard_tactics_pi#92]
This is for the record, some change has made that while the Dashboard (the buggy one) still builds OK it does not launch, at least on Windows is happily crashes after it has retrieved all its identification strings and the PNG-bitmap! Regression...

**Comment by canne** [comment]
This is annoying, me who wanted to get rid off this code, I can't leave die like this... 😀

**Comment by canne** [comment]
Compiled on Raspberry Pi 4B (4GB) Dashboard: core dumps.

**Comment by canne** [comment]
Fixed with b0eaf632a1964c3274bbd3bf261ecef4e4473299

Actually his not regression, I remove the tag: I have never before removed the ini-file to have a fresh nice start of the OpenCPN with Dashboard.

I just wonder how Dashboard can never worked in a fresh installation or in case when user clears the ini-file? Let's look a this, quite curious. In LoadConfig(), if no instruments has been defined (we just removed the ini-file to get a good fresh start,  a default list is given to the user:
```
                // This is the default instrument list
                ar.Add( ID_DBP_I_POS );
                ar.Add( ID_DBP_D_COG );
                ar.Add( ID_DBP_D_GPS );
            }
```
After one second from the start of the object, hits Notify() event. It wants to set the UTC time to all instruments (the time is invalid at this point, but that is not the problem). We end up to
DashboardWindow::SendUtcTimeToAllInstruments() - which I have modified for DashT code already - because it needs the uint64 sentence identifier like everywhere else. But in Dashboard part the code was crazy, here are my corrections:
```
             // ( m_ArrayOfInstrument.Item( i )->m_cap_flag & OCPN_DBP_STC_GPS ) // default _D_GPS has no SetUtcTime()!
             ( m_ArrayOfInstrument.Item( i )->m_cap_flag & OCPN_DBP_STC_CLK )
             && m_ArrayOfInstrument.Item( i )->m_pInstrument->IsKindOf( CLASSINFO( DashboardInstrument_Clock ) ) )
```
Explanation: if we keep sending the UTC time to instruments to _STC_GPS sentence subscribed items, we end up to cast-force a call to `SetUtcTime()` method on a `DashboardInstrument_GPS` class object we just created. But there is no `SetUtcTime()` method in there!  _Et voilà_! Segmentation fault! Bravo. This must be some old code never really tested. Also the usage of `IsKindOf()` method as a selector (contradictory selector, that is), is a joke since it return true for practically all instrument classes since they have the same base class.

Closing now.
